### ISSUE TITLE
How to organize blog file to subdirectories

### ISSUE THREAD

**Issue by yuemeng200** [satnaing/astro-paper#366]
the slug can't include /

**Comment by satnaing** [comment]
Yes, slug cannot include `/` as it will create another route segment.
Are you trying to achieve something like this `example.com/blog/category/post-title`?

**Comment by JesseZhuang** [comment]
Interested. Would be nice to be able to organize the markdown files with folders.

**Comment by tyayers** [comment]
I am also interested in this, I would like to organize posts by year sub-folders.

**Comment by satnaing** [comment]
Fixed in #481
